### ISSUE TITLE
Initiation not working

### ISSUE THREAD

**Issue by reg-is** [adamkempenich/magichome-python#3]
In the **readme** file, it is writen to use the `MagicHomeAPi()` method, which isn't defined. Shouldn't it be the `__init__()` method instead?

For `__init__(self, device_ip, device_type, keep_alive=True)`, what am I suposed to enter for `self` and `keep_alive=True`?

UPDATE: Sorry, I never worked with Python before. I found the solution for my problem now

**Comment by adamkempenich** [comment]
Sorry to not get back to this! If anyone else is unfamiliar with Python and happens upon this, _self_ carries information about each device into each of the functions. 
_keep/_alive=true_ fixes a bug in the MagicHome devices, where a device will 'sleep' if it doesn't receive any communication after an extended amount of time. If you don't want this functionality, you can override it by setting your third function parameter to _false_. 

TL;DR, no values are needed for _self_ and _keep/_alive_
